### ISSUE TITLE
Test fail with latest jaxtyping (0.2.29)

### ISSUE THREAD

**Issue by GaetanLepage** [patrick-kidger/equinox#740]
Since the last release of jaxtyping, two tests of the equinox test suite fail:

```
=========================== short test summary info ============================
FAILED tests/test_errors.py::test_traceback_runtime_eqx - assert 4 == 1
FAILED tests/test_errors.py::test_traceback_runtime_custom - assert 7 == 4
============ 2 failed, 389 passed, 1 skipped, 37 warnings in 41.82s ============
```

**Comment by GaetanLepage** [comment]
Full logs: https://pastebin.com/hDHNzj7t

**Comment by patrick-kidger** [comment]
Thanks for the report! Looks like a harmless failure, and we just need to tweak the tests to be a bit more robust. Fixed in #742. :)
